.vscode/settings.json

```diff
@@ -8,7 +8,7 @@
   },
   "editor.codeActionsOnSave": {
     "source.fixAll.eslint": true,
-    "source.organizeImports": true
```

Comment:
Can you comment for what reason you have removed this setting? I find it useful to automatically organize imports on save. Sometimes it removes unused imports which you are going to use, and that causes some inconvenience. However if we remove this, then we will just forget to trigger this source action manually, which will lead to chaos in imports.

Reply:
I was noticing that my imports were being automatically ordered alphabetically and I thought that ```import React from "react"``` had to be at the top. I was also troubleshooting an issue and changed this value to ```false``` as part of troubleshooting steps; I didn't change it back to ```true``` after resolving my troubleshooting. 

I didn't realize that my local vscode settings would be included in a commit. I am willing to accept any pull that includes modifications to vscode settings.

Reply:
The point is that you have modified the workspace settings, which are stored in the repository in `.vscode/settings.json`. These settings are supposed to be committed, but the changes should be reasonable. That is why I request to use any graphical GIT tool like the one included into VSCode, because it allows to inspect every change that you are going to commit. Please read the “Don't include irrelevant changes into commits.” section of the README file.